docs/member/strategist-profile.md

```diff
@@ -0,0 +1,64 @@
+# Strategist Profile
+
+## Introduction
+AlgoBulls provides a variety of trading strategies, but the brains behind these strategies are market experts who work extremely hard to create complex yet simple strategies. The Strategist profile will help you recognize the strategist's work and strategies. Let’s take a look at what the strategist profile offers
```

Comment:
`create complex yet simple strategies`

@pushpakdagade i understand what we are trying to convey here, but there is something wrong with the usage of these terms like this. please help to change this

Reply:
`Let’s take a look at what the strategist profile offers`

there should have been a full stop (.) or colon (:) here at the end

Reply:
Full stop added

Reply:
@akhil-jain-algobulls yes the language doesn't seem proper. Inform the same to Mayuri and get her to revise the language.

Reply:
`create complex yet simple strategies` @hemendra-sakpal-algobulls please get this part revised

Reply:
@akhil-jain-algobulls, the updated content is pushed, please check